templates/openwisp2/settings.py

```diff
@@ -359,6 +375,20 @@
 RAVEN_CONFIG = {{ openwisp2_sentry|to_nice_json }}
 {% endif %}
 
+{% if openwisp2_monitoring %}
+TIMESERIES_DATABASE = {
+    'BACKEND': '{{ openwisp2_timeseries_database.backend }}',
+    'USER': '{{ openwisp2_timeseries_database.user }}',
+    'PASSWORD': '{{ openwisp2_timeseries_database.password }}',
+    'NAME': '{{ openwisp2_timeseries_database.name }}',
+    'HOST': '{{ influxdb_http_ip }}',
+    'PORT': '{{ influxdb_http_port }}',
+}
+
+INSTALLED_APPS.append('djcelery_email')
```

Comment:
I still think we should add a dedicated setting for enable `djcelery_email` @nemesisdesign 
We can use that to install `djcelery_email` and add needed settings. 
It will also allow us to move ahead with https://github.com/openwisp/openwisp-monitoring/pull/254

It will also open possibility to add a dedicated queue for emails. What do you think about this?

Reply:
@pandafy what is the use case for that? Why should we do this additional effort, what benefit do we get back from it?
I feel that this form is enough to do what we and most users need and if anyone needs to override this they can do it, but I don't see a good reason to change this part. Sending emails in the background is always a good idea.

Reply:
If we set `openwisp2_email_backend` to use djcelery_email backend as default and do not install the package by default, that will make users run into errors. 

We should install that package by default and not just limit it to be used when openwisp-monitoring is enabled.

Reply:
ok @pandafy but what you ask is a different kind of change (make django-celery-email the default email backend) and not strictly related to openwisp-monitoring? I think how it was before was good enough for introducing openwisp-monitoring to ansible-openwisp2, which is the goal we're interested in this PR.

I suggest simply reverting https://github.com/openwisp/ansible-openwisp2/pull/262/commits/59796a1e231670bffeb084e5a890fda4fc64a3b8 and create a new issue to make the django-celery-email backend the default one.

Reply:
ok @nemesisdesign, opened https://github.com/openwisp/ansible-openwisp2/issues/271